src/ships.js

```diff
@@ -0,0 +1,33 @@
+function Ship(itinerary) {
```

Comment:
I think it would be more appropriate to use a constructor here, so all the logic could be contained in one space. It would still accept the itinerary as here, and it would save having to call the prototype method

Also, is there a reason the filename is ships when the class is `Ship`?